readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -248,7 +248,7 @@ def build(self):
             '--site-dir',
             self.build_dir,
             '--config-file',
-            self.yaml_file,
+            os.path.relpath(self.yaml_file, self.root_path),
```

Comment:
One test was expecting this call to be relative to the checkout path, but with the new change this was a full path.
Relative path is shorter to show in the output and we are already setting the cwd to the checkout path, so we are fine.